input/fsh/ConsentSJF.fsh

```diff
@@ -0,0 +1,75 @@
+Profile: ConsentSJF
+Parent: Consent
+Id: ConsentSJF
+Description: "Profile to represent a consent to shared record keeping in between care providers in Sweden."
+
+* ^purpose = "To enable creating consents, evaluation and exchange of already stored consents"
+
+* ^version = "1.0.0"
+* ^status = #active
+
+* identifier ..1
+
+* scope.coding 1..1
+* scope.coding.system 1..
+* scope.coding.system from ConsentScopeCodes (extensible)
+* scope.coding.system ^binding.description = "Consent Scope Codes"
+* scope.coding.code 1..
+* scope.coding.code = #patient-privacy (exactly)
+
+* category ..1
+* category.coding 1..1
+* category.coding.system 1..
+* category.coding.system = SCT (exactly)
+* category.coding.version = "http://snomed.info/sct/45991000052106/version/20220531" (exactly)
+* category.coding.code 1..
+* category.coding.code = #70831000052108 (exactly)
+
+* patient 1..
+* patient only Reference(PatientSEVendorLite)
+
+* dateTime 1..
+
+* policy 1..1
+* policy.uri 1..
+* policy.uri = "https://www.riksdagen.se/sv/dokument-lagar/dokument/svensk-forfattningssamling/patientdatalag-2008355_sfs-2008-355" (exactly)
+
+* provision 1..
+* provision.type 1..
+* provision.type = #permit (exactly)
+* provision.period 1..
+* provision.actor ^slicing.discriminator.type = #value
+* provision.actor ^slicing.discriminator.path = "role.coding.code"
+* provision.actor ^slicing.description = "Slices for different actor roles"
+* provision.actor ^slicing.ordered = false
+* provision.actor ^slicing.rules = #closed
+* provision.actor contains
+    grantor 1..1 MS and
+    grantee 1..1 MS and
+    primaryauthor 0..1 MS
+
+* provision.actor[grantor].role.coding.system = ROLE_CODE_SYSTEM (exactly)
```

Comment:
ROLE_CODE_SYSTEM is probably an alias that has not been included in the original commit
Either add ROLE_CODE_SYSTEM with correct URI to the aliases.fsh file or replace with correct value directly

Reply:
Fixed that in the branch "28-talk-to-hl7-sweden..."